packages/playground/ssr-framework/vue-framework/build.js

```diff
@@ -0,0 +1,37 @@
+const { getViteConfig } = require('./getViteConfig')
+const hydrateFile = require.resolve('./hydrate')
+const getPagesFile = require.resolve('./getPages')
+
+module.exports.build = build
+
+async function build(root, silent) {
+  const { build } = require('vite')
```

Comment:
**suggestion (dx):** You are shadowing the function itself
You could use `const { build: viteBuild } = require('vite')` to rename the inner build and then use it in line 16 and 27

Reply:
👍 Done